module/zstd/zfs_zstd.c

```diff
@@ -50,7 +50,14 @@
 #include "lib/zstd.h"
 #include "lib/common/zstd_errors.h"
 
-static kstat_t *zstd_ksp = NULL;
+static int zstd_lz4_pass = 1;
+static int zstd_cutoff_level = ZIO_ZSTD_LEVEL_3;
+static int zstd_zstd_pass = 1;
+static int zstd_hard_moed = 0;
+static unsigned int zstd_abort_size = (131072);
+
+kstat_t *zstd_ksp = NULL;
```

Comment:
I don't see why this needs to not be static anymore?

Reply:
It doesn't, it's just a change that I overlooked when merging.

Reply:
Fixed, unless I'm worse at reading than I think I am.